fastlane/lib/fastlane/actions/ensure_git_status_clean.rb

```diff
@@ -7,7 +7,13 @@ module SharedValues
     # Raises an exception and stop the lane execution if the repo is not in a clean state
     class EnsureGitStatusCleanAction < Action
       def self.run(params)
-        repo_status = Actions.sh("git status --porcelain")
+        if params[:ignore_file] 
+          file_for_ignore = params[:ignore_file]
```

Comment:
The key you are fetching from `params` needs to be defined in the `options` method below 😇 

Can you add the option in but also rename this option to `:ignored` so that it matches the CLI name? 😊

Reply:
I modified `ignore_file` to `ignored` and I added `params` from `options`